_commands/basics/jobs.md

```diff
@@ -3,22 +3,99 @@
 
 jobs
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+
+`jobs` is used to list the jobs running in the background of the current shell session.
+
+The general format is
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ jobs [options] jobID
 ~~~
 
 <!--more-->
 
+A simple example is:
+
+~~~ bash
+$ gnome-calculator &
+[1] 14616
+$ xeyes &
+[2] 14624
+$ jobs
+[1]-  Running                 gnome-calculator &
+[2]+  Running                 xeyes &
+~~~
+
+This will display the status of every job in the current shell.
```

Comment:
how does it display the status? is it just by the presence/lack of the `&`, or is the `-`/`+` also important as well?